floss/main.py

```diff
@@ -313,17 +315,15 @@ def print_all_strings(path, n=4, quiet=False):
             print("%s" % (s.s))
     else:
         print("Static ASCII strings")
-        print("Offset       String")
-        print("----------   -------------------------------------")
-        for s in strings.extract_ascii_strings(b, n=n):
-            print("0x%08X   %s" % (s.offset, s.s))
+        print(tabulate.tabulate(
+            [("0x%X" % (s.offset,), s.s) for s in strings.extract_ascii_strings(b, n=n)],
```

Comment:
this is ugly, sorry. but we need a list of 2-tuples containing formatted strings. its going to be messy no matter how you do it...